### PR TITLE
Added 22 unique English quotes 

### PR DESCRIPTION
**Pull request by ROCKRAJA949**
### Description
Added 22 unique quotes with ID's ranging from 6881-6903.

### CODE REVIEW

**Comment by Miodec** [comment]
There was a different PR with english quotes opened before this one. Please merge master in and resolve ID conflicts.

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -39012,6 +39012,144 @@
...
+      "id": 6881
+    },
+    {
+      "text": "The hour of departure has arrvied, and we go our ways. I to die, and you to live. Which is better? God only knows.",
+      "source": "Socrates",
+      "length": 114,
+      "id": 6882
+    },
+    {
+      "text": "Of all sad words of tongue or pen, the saddest are these, 'It might have been.'",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39029]: Change `'` to `\"`

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -39012,6 +39012,144 @@
...
+      "id": 6894
+    },
+    {
+      "text": "The weak fear happiness itself. They can harm themselves on cotton wool. Sometimes they are wounded even by happiness.",
+      "source": "Osamu Dazai, No Longer Human",
+      "length": 118,
+      "id": 6895
+    },
+    {
+      "text": "When the axe came into the woods, many of the trees said: 'At least the handle is one of us.'",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39107]: Same thing here, use `"`

File: frontend/static/quotes/english.json [hunk 3]
```diff
@@ -39012,6 +39012,144 @@
...
+      "id": 6900
+    },
+    {
+      "text": "I find that the stranger life gets, the more it seems to make sense.",
+      "source": "Ivern, League of Legends",
+      "length": 68,
+      "id": 6901
+    },
+    {
+      "text": "I touched the stars, and saw the glorious light og a thousand suns! Now, blinded by this elegance, how could my purpose be anything... but dark...",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39143]: `og` -> `of`

File: frontend/static/quotes/english.json [hunk 4]
```diff
@@ -39012,6 +39012,144 @@
...
+      "source": "Aatrox, League of Legends",
+      "length": 146,
+      "id": 6902
+    },
+    {
+      "text": "Stand Amongst The Ashes Of A Trillion Dead Souls And Ask The Ghosts If Honor Matters. The Silence Is Your Answer.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39149]: Every word is capitalised.

File: frontend/static/quotes/english.json [hunk 5]
```diff
@@ -39012,6 +39012,144 @@
...
+      "source": "Ned Vizzini, It's Kind of a Funny Story",
+      "length": 215,
+      "id": 6894
+    },
+    {
+      "text": "The weak fear happiness itself. They can harm themselves on cotton wool. Sometimes they are wounded even by happiness.",
+      "source": "Osamu Dazai, No Longer Human",
+      "length": 118,
+      "id": 6895
+    },
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39105]: Bit too depressing imo. Please remove.

**Comment by ROCKRAJA949** [comment]
Give me a week, Going through finals, will make requested changes soon

**Pull request closed without merging**